preparation.py

```diff
@@ -60,3 +77,12 @@ def preprocess_image(img, target_size=(128, 128)):
     else:
         x = torch.autograd.Variable(x, volatile=True)
     return x
+
+
+def add_dropped_objects(original, processed):
+    reconstructed = processed
```

Comment:
@apyskir it is usually safer to make a copy with .copy() or something as this may result in changing both objects later not just one.

Reply:
@jakubczakon Ok, I'll change it.